src/Symfony/Component/Console/Command/AlarmableCommandInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Interface for command that dispatches SIGALRM signal.
```

Comment:
Is the command actually dispatching SIGALRM signals ? The rest of the code gives me the impression that it *listens* to those signals.

Reply:
Fixed